ee/clickhouse/queries/funnels/test/test_funnel_correlation.py

```diff
@@ -160,6 +171,104 @@ def test_basic_funnel_correlation_with_events(self):
         self.assertEqual(len(self._get_people_for_event(filter, "negatively_related", success=False)), 5)
         self.assertEqual(len(self._get_people_for_event(filter, "negatively_related")), 0)
 
+    @snapshot_clickhouse_queries
+    def test_funnel_correlation_with_events_and_actions(self):
+
+        for i in range(10):
```

Comment:
Is there a reason for so many events? If not, I would suggest we go with the minimum reproducible test case for this specific test.

Reply:
Fair, will update!